.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```

Comment:
what is the reason for adding this?

Reply:
hmm I didn't add this. It seems like it was automatically added when I create the branch

Reply:
Looking at the master branch, it seems like we actually don't have a .gitattributes file right now, so this would be adding a new file. So, if you can just delete this file from the PR alltogether, that should be fine